test/test_functionalization.py

```diff
@@ -272,7 +273,7 @@ def f(x):
             x.transpose_(1, 0)
             y = x[0]
             y.add_(tmp)
-            return y
+            return x
```

Comment:
does the return here matter?

Reply:
yep; first I got a repro from LTC tests, then I tried to swizzle this test until I could get a repro from it.

I also added `self.assertEqual(out_ref.size(), out_functional.size())` to `assert_functionalization` above.

This version of the test now fail the new assert above without the change, because the `FunctionalTensorWrapper` for x used to have its sizes set incorrectly.